app/src/main/java/com/codesoom/assignment/resources/ResourceFactory.java

```diff
@@ -0,0 +1,24 @@
+package com.codesoom.assignment.resources;
+
+import com.codesoom.assignment.HttpMethod;
+import com.codesoom.assignment.models.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ResourceFactory {
+
+    public TaskResource createResource(HttpMethod method) {
+        TaskResource taskResource = null;
+        List<Task> tasks = new ArrayList<>();
+        switch (method) {
+            case GET:
+                taskResource = new GetTaskResource();
```

Comment:
이것으로 충분하지 않을까요? 어째서 할당을 해야 하나요?
```suggestion
                return new GetTaskResource();
```

Reply:
이렇게 세세하게 봐주셔서 감사합니다. 뭔가 제가 급하게 날코딩을 하는 것을 잡아주는 느낌이라 도움이 됩니다. 
YAGNI 원칙이 생각이 납니다. 제가 가지고 있는 좋지 않은 습관인 것 같은데 이번 기회에 잘 고쳐보겠습니다.
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it